### ISSUE TITLE
Adding a hyperlink always adds extra space

### ISSUE THREAD

**Issue by Nobody109** [agusmakmun/django-markdown-editor#59]
When using the toolbar button for adding an hyperlink it always includes an extra whitespace directly after the hyperlink.

Attaching 2 screenshots which show the issue.

![text_before_applying_hyperlink](https://user-images.githubusercontent.com/1160957/51604175-3e7da100-1f0c-11e9-9c81-a77b06df0724.png)
![extra_space_after_hyperlink](https://user-images.githubusercontent.com/1160957/51604177-3e7da100-1f0c-11e9-8748-3b729618c1bf.png)

**Comment by agusmakmun** [comment]
Yes, I'm not indentify the next column is a character or white space. Did you have any solution for this?
For the [source can be found here..](https://github.com/agusmakmun/django-markdown-editor/blob/master/martor/static/martor/js/martor.js#L446)

```
editor.session.replace(range, '['+text+'](http://) ');
```

**Comment by some1ataplace** [comment]
Maybe try this. Did not test this. 

1. Add a custom JavaScript function to trim the link:

function trimLink(link) {
  return link.trim();
}


2. Modify the "insertLink" function in the Markdown Editor to call the "trimLink" function:

function insertLink() {
  var link = prompt("Enter the link URL:");
  if (link) {
    link = trimLink(link);
    var text = prompt("Enter the link text:");
    if (text) {
      text = trimLink(text);
      var selection = getSelection();
      var range = selection.getRangeAt(0);
      var node = document.createElement("a");
      node.setAttribute("href", link);
      node.textContent = text;
      range.insertNode(node);
    }
  }
}
